contents/blog/dev-marketing-paid-ads.md

```diff
@@ -0,0 +1,154 @@
+---
+
+Since starting PostHog in 2020, we’ve learned a bunch about what does and doesn’t work when it comes to [marketing to engineers](/blog/dev-marketing-for-startups). Paid ads is a slightly scary area because it is _very_ easy to waste a bunch of money doing stuff that doesn't work. And a lot of it doesn't work. 
+
+This article shares specific, actionable advice for early-stage startups building dev tools in particular. I'm assuming that you've already decided that running paid ads is something that makes sense for your company. 
+
+I've split this into two sections - general misconceptions and then what we've learned, channel-by-channel. 
+
+## Clearing up some misconceptions
+
+### Paid ads =/= marketing
+
+Paid ads are a _subset_ of marketing. Every time you talk about your product in _any_ kind of context, you are doing marketing. This includes your website, turning up to events, writing content, posting on Twitter, chatting in an external Discord group, creating a YouTube tutorial, etc.
+
+This means taking paid ads seriously (because of the $$$), but don't spend all your time here. At PostHog, paid ads take up ~10% of our marketing team's time in an average week. We spend 80%+ on writing. 
+
+### All paid ads are basically the same
+
+Very crudely, your ads will have one of two objectives - _conversion_ or _awareness_. 
+
+Conversion means that you want people to click on your ad and then do something like sign up for your product. You can define conversion how you like - this is just a typical example. Conversion is easy to measure, which is why marketers like it. However, you are generally only _capturing_ demand from people already aware of your product. 
+
+Awareness means you want people to see your ad. Maybe they won't click on it - that's ok. Your objective is to just get the word out so that someone is exposed to your product enough times that at some point they Google 'super cool product' and come sign up. Awareness-based ads are a small part of _creating_ demand. This takes more effort to measure, but is totally possible - see below.  
+
+We split our budget 50-50 between the two. This is not a 'rule' - just don't spend it all on conversion. 
+
+### Paid ads don't work on developers
+
+Everyone who works at a tech company basically thinks they are immune to ads because they are super good at internetting, and have never seen an ad for a new product, clicked on it, and then signed up. Your _quantitative_ attribution data in PostHog or whatever analytics tool you are using will seem to back this up. 
+
+This is why we ask all users where they heard about PostHog whenever they sign up or book a demo - it's a simple (optional) free text field. Enough of our users say 'ad on Google' or similar that we know paid ads do actually reach a large chunk of them. Especially ones with money to spend on behalf of their company. However, you need to take the time to collect and digest this _qualitative_ attribution data in the first place. 
```

Comment:
Add screenshot of this field on our sign up page here?

Reply:
Would prefer not to if that's ok (I think people will get it, and also I can't abide having a single orphan image in an otherwise all-text article...)

Reply:
Yup, fair